usr/bin/sandbox-app-launcher

```diff
@@ -207,6 +206,12 @@ setup_or_check() {
     run_if_root chmod 1777 "${shared_dir}"
   fi
 
+  ## Check if the second wrapper script exists.
+  if ! [ -f "${wrapper_script}-wx" ]; then
+    echo "ERROR: File '${wrapper_script}-wx' does not exist. This package was not installed properly." >&2
+    exit 1
+  fi
```

Comment:
Then this test could be avoided. Exitence would be safe to presume.

Reply:
The test would still be required. We cannot guarantee that the user has even, e.g. installed the script via the Debian package as they may be attempting to use the script on another distribution. Better to fail hard than cause a lot of confusion.

Reply:
OK. Well, there's lots of other files who's existence we're not checking. I was wondering about that too but usually as far as I can see in other scripts, they assume that scripts that need to be shipped in the package are available. If these are many files, then probably best to use a list to loop over.

Reply:
I'm not referring to files actually within the repository; only those created by the postinst script since non-Debian users won't have those. We check for the existence of nearly all files/directories dynamically generated by the Debian package upon installation:

* /usr/share/sandbox-app-launcher/
* /var/cache/sandbox-app-launcher-autogenerated/
* /home/sandbox-app-launcher-appdata/
* /usr/share/sandbox-app-launcher/wrapper-script-wx

The only things we don't check for are both seccomp filters which we could add checks for also.

Reply:
> I'm not referring to files actually within the repository; only those created by the postinst script since non-Debian users won't have those.

Great factor. Makes sense.